docs/platforms/github-com.md

```
@@ -0,0 +1,111 @@
+![How CodeRabbit Works](/img/integrations/how-codeRabbit-works.png)
+
+## Let's Integrate CodeRabbit with GitHub
+
+### 1. Accessing CodeRabbit
+
+Visit the [Code Rabbit login](https://app.coderabbit.ai/login?free-trial) page. You'll see all the authentication options, select "**Login with GitHub**" to proceed.
+
+![CodeRabbit Login](/img/integrations/login-self-hosted-github.png)
+
+### 2. Authorization
+When you choose **Login with GitHub** in step1 , you'll be prompted to authorize CodeRabbit. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)